workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -46,20 +70,40 @@ protected override void OnCreateManager(int capacity)
             injector = new RequiredFieldInjector(entityManager, logger);
         }
 
+
+        protected override void OnUpdate()
+        {
+            for (var i = 0; i < addedEntitiesData.Length; i++)
+            {
+                var entity = addedEntitiesData.Entities[i];
+                CreateActivationManagerAndReaderWriterStore(entity);
+                PostUpdateCommands.AddComponent(entity, new GameObjectReferenceHandle());
```

Comment:
Change to `PostUpdateCommands.AddComponent(entity, ComponentType.Create<GameObjectReferenceHandle>())`